master/propidx.html

```diff
@@ -210,9 +210,7 @@ <h1>Property Index</h1>
         </tr>
         <tr>
           <th><a>'stop-color'</a></th>
-          <td>currentColor |<br />
-           <a>&lt;color&gt;</a>
-           [<a>&lt;icccolor&gt;</a>]</td>
+          <td>&lt;‘<a>'color'</a>’&gt;</td>
```

Comment:
I don't see a mention that we are dropping `icc()` in the SVG2 change log. Do you know if we have a resolution for that? We probably have.

Reply:
I don't know if there is a resolution. This edit simply makes stop-color consistent with other color properties.

The spec is currently inconsistent as it suggests `#A8F` and `white` are valid stop-color values, but the grammar requires something like`<color> icc-color(<name> [ ',' <number> ]+ )`.